beacon-chain/forkchoice/doubly-linked-tree/store_test.go

```diff
@@ -79,7 +79,7 @@ func TestForkChoice_HasNode(t *testing.T) {
 func TestStore_Head_UnknownJustifiedRoot(t *testing.T) {
 	f := setup(0, 0)
 
-	f.store.justifiedCheckpoint.Root = [32]byte{'a'}
+	f.store.justifiedCheckpoint = &forkchoicetypes.Checkpoint{Epoch: 1, Root: [32]byte{'a'}}
```

Comment:
Why not epoch 0? Is it inconsequential for this test, or does it fail if 0?

Reply:
Epoch zero is special in that I completely disregard the root and use zerohash if the justified epoch is genesis since the only possible justified checkpoint can be the genesis block root